data/tabular/bicerano_dataset/transform.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import wget
+
+def transform_data():
+
+
+    original_data = pd.read_excel("HT_MD_polymer_properties.xlsx")
+    
+    clean_data = original_data.drop("sl_num", axis=1)
+    assert not clean_data.duplicated().sum()
+
+    clean_columns = ["Name", 
+                     "SMILES", 
+                     "Tg_exp", 
+                     "Tg_calc", 
+                     "Tg_calc_std", 
+                     "rho_300K_exp", 
+                     "rho_300K_calc",
+                     "rho_300K_calc_std", 
+                     "glass_CTE_calc", 
+                     "glass_CTE_calc_std", 
+                     "rubber_CTE_calc", 
+                     "rubber_CTE_calc_std"]
+    
+    clean_data.columns = clean_columns
+ 
+ 
+    clean_data["SMILES"] = clean_data["SMILES"].str.replace("[Ce]", "[*]", regex=False)
+    clean_data["SMILES"] = clean_data["SMILES"].str.replace("[Th]", "[*]", regex=False)

```

Comment:
Ok, it is not a plain monomer SMILES. 

so I'd suggest two things: 

1. Add the `PSMILES` type in the data schema
2. Add the monomer SMILES (adding back leaving groups) 
3. Add templates that make clear that those are homopolymers (?) with this polymer SMILES and/or monomer SMILES 
4. You can add additional templates for other tasks such as conversational prompts "give me a PSMILES with density X and Tg Y"